packages/components/src/combobox-control/README.md

```diff
@@ -116,6 +116,13 @@ The current value of the input.
 -   Type: `mixed`
 -   Required: Yes
 
+#### looseSearch
+
+If present, search matching is made less restrictive by removing spaces and special characters.
+
+-- Type: `Boolean`
+-- Required: No
```

Comment:
Please change it with the below code. Your changes had an extra `-` dash and it was not consistent with the existing markdown 🙂 

```suggestion
-   Type: `Boolean`
-   Required: No
```

Reply:
Done!